coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -43,6 +43,10 @@
         'file_arg_template': '+{line},{column} {filename} ',
         'gui': False
     },
+    'emacs': {
+        'file_arg_template': '+{line}:{column} {filename}',
+        'gui': False
+    },
```

Comment:
AKAIK emacsclient opens a file in an already running instance of emacs. We dont want that.